tests/scheduler/scheduler.test.ts

```diff
@@ -0,0 +1,48 @@
+import { Scheduled, TestingRuntime, Workflow, WorkflowContext } from "../../src";
+import { DBOSConfig } from "../../src/dbos-executor";
+import { createInternalTestRuntime } from "../../src/testing/testing_runtime";
+import { sleep } from "../../src/utils";
+import { generateDBOSTestConfig, setUpDBOSTestDb } from "../helpers";
+
+describe("scheduled-wf-tests", () => {
+    let config: DBOSConfig;
+    let testRuntime: TestingRuntime;
+  
+    beforeAll(async () => {
+        config = generateDBOSTestConfig();
+        await setUpDBOSTestDb(config);  
+    });
+  
+    beforeEach(async () => {
+        testRuntime = await createInternalTestRuntime([DBOSSchedTestClass], config);
+    });
+  
+    afterEach(async () => {
+        await testRuntime.destroy();
+    }, 10000);
+  
+    test("wf-scheduled", async () => {
+        await sleep(3000);
+        expect(DBOSSchedTestClass.nCalls).toBeGreaterThanOrEqual(2);
+        expect(DBOSSchedTestClass.nTooEarly).toBe(0);
+        expect(DBOSSchedTestClass.nTooLate).toBe(0);
+    });
+});
+
+class DBOSSchedTestClass {
+    static nCalls = 0;
+    static nTooEarly = 0;
+    static nTooLate = 0;
+
+    // eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
We can remove this eslint-disable if we return `Promise.resolve()` from the workflow.

Reply:
Not only that, but it has an await() :-D